frontend/public/models/yaml-templates.ts

```diff
@@ -767,4 +767,43 @@ spec:
     delayAfterAdd: 10s
     delayAfterDelete: 10s
     delayAfterFailure: 10s
+`).setIn([referenceForModel(k8sModels.VirtualMachineModel), 'default'], `
+apiVersion: ${k8sModels.VirtualMachineModel.apiGroup}/${k8sModels.VirtualMachineModel.apiVersion}
+kind: VirtualMachine
+metadata:
+  name: example
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/size: small
+    spec:
+      domain:
+        devices:
+          disks:
+            - name: registrydisk
+              volumeName: registryvolume
```

Comment:
volumeName was removed in latest kubevirt so please remove it too and volume needs to have the same name as disk so disk/name = registrydisk  => volume/name = registrydisk too. Same applies to cloudinitdisk